pkg/cmd/roachprod/flags.go

```diff
@@ -490,4 +491,6 @@ func initFlags() {
 		"time-range", []int64{}, "grafana annotation time range in epoch time")
 	fetchLogsCmd.Flags().DurationVarP(&fetchLogsTimeout,
 		"timeout", "t", 5*time.Minute, "Timeout for fetching the logs from the cluster nodes")
+	pprofTimeBefore = getLatestPProf.Flags().StringP(
```

Comment:
Question around the flag: Is there a specific reason we deviate from using the `Var` variant used throughout for flags and not passing a &(str) pointer for `pprofTimeBefore`?

Reply:
I wanted to use the shorthand which was not available for StringVar